libjxl/build.gradle

```diff
@@ -0,0 +1,62 @@
+plugins {
+    id 'com.android.library'
+    id 'org.jetbrains.kotlin.android'
```

Comment:
It would be better to not use kotlin in the library, as it will add kotlin dependenciesto non-kotlin apps.